base.js

```diff
@@ -48,7 +48,7 @@ module.exports = {
     'array-func/prefer-flat': 'off', // https://github.com/freaktechnik/eslint-plugin-array-func#prefer-flat
     'array-func/prefer-flat-map': 'warn', // https://github.com/freaktechnik/eslint-plugin-array-func#prefer-flat-map
     'arrow-body-style': ['error', 'as-needed'], // https://eslint.org/docs/rules/arrow-body-style
-    'arrow-parens': ['error', 'as-needed'], // https://eslint.org/docs/rules/arrow-parens
+    'arrow-parens': ['error', 'always'], // https://eslint.org/docs/rules/arrow-parens
```

Comment:
are you sure arror-parens: always doesn't cause other issues? i'm thinking about simple arrow functions here.
something like: array.map(x => x+1) should not require extra parens.
I don't know if that rule does that though. Could you verify that?

Reply:
You are right in the map function it would require parens for the parameter `x`. I think that's ok and more consistent. Think about adding another parameter like `index` the second of the map function. Than we would have to add the parentheses anyways. 

Also when you think about destructuring in function parameters:
```javascript
myArray.map((data) => data.name)
myFunc(({}) => name)
```

As you can see having the parentheses seems more consistent to me so I would enforce them always. 

Have I convinced you?

Reply:
yeah, i was just making sure that we thought about that case as well.